src/components/content/deploymentVariables/DeploymentVariables.tsx

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
this must not be a new sub menu. It must be a table similar to service policies.